rosdep/python.yaml

```diff
@@ -6515,6 +6515,22 @@ python3-flask-cors:
   ubuntu:
     '*': [python3-flask-cors]
     bionic: null
+python3-flask-jwt-extended:
+  debian:
+    bullseye: [python3-python-flask-jwt-extended]
+    pip:
```

Comment:
This should explicitly call out `buster`. Right now it's resolved as a pip version.

Reply:
resolved in bf1169b2bc4b36c3c5fb44d670d279a9a62bee5d